src/containers/appointmentPicker/AppointmentPicker.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+
+import { ProviderPicker } from '../../components/providerPicker/ProviderPicker';
+
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+import { TimePicker } from '../../components/timePicker/TimePicker';
+
+import { getAvialableTimes } from '../../data';
```

Comment:
nit: typo. Your IDE should have a spell checker. If not then probably an extension.